### DISCUSSION TITLE
How to puppeteer + Motion Canvas?

### DISCUSSION THREAD

**Discussion by sdproyi** [motion-canvas/motion-canvas#928] [Q&A]
I have no idea what I'm doing wrong or right because I get no errors if u know how to make puppeteer work in motion canvas let me know.

**Comment by aarthificial** [comment]
Our E2E package runs Motion Canvas using puppeteer, maybe it can help:
https://github.com/motion-canvas/motion-canvas/tree/main/packages/e2e
